src/Components/Authentication/__tests__/Desktop/Components/DesktopModal.test.tsx

```diff
@@ -1,9 +1,7 @@
 import { mount } from "enzyme"
 import React from "react"
-import {
-  CloseButton,
-  DesktopModal,
-} from "../../../Desktop/Components/DesktopModal"
+import Icon from "../../../../Icon"
+import { DesktopModal } from "../../../Desktop/Components/DesktopModal"
```

Comment:
We have the `src` directory set in jest's [moduleDirectories](https://jestjs.io/docs/en/configuration#moduledirectories-array-string) setting, so we shouldn't have to do relative imports of these components. For icon it could be 

```typescript
Import icon from "Components/Icon"
```

Reply:
Oh cool -- had some issues in the past with absolute paths in jest, but seems to run ok now.  Vscode still gives me errors, maybe there's a local config I'm missing?